modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessorTestCase.java

```diff
@@ -56,61 +56,46 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
+import io.qameta.allure.Issue;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
 import org.junit.After;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
-
-import org.hamcrest.BaseMatcher;
-import org.hamcrest.Description;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
 
 public class ComponentMessageProcessorTestCase extends AbstractMuleContextTestCase {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ComponentMessageProcessorTestCase.class);
   @Rule
   public ExpectedException expected = ExpectedException.none();
-  private ComponentMessageProcessor<ComponentModel> processor;
-  private ExtensionModel extensionModel;
-  private ComponentModel componentModel;
-  private ResolverSet resolverSet;
-  private ExtensionManager extensionManager;
-  private PolicyManager mockPolicyManager;
+  protected ComponentMessageProcessor<ComponentModel> processor;
+  protected ExtensionModel extensionModel;
+  protected ComponentModel componentModel;
+  protected ResolverSet resolverSet;
+  protected ExtensionManager extensionManager;
+  protected PolicyManager mockPolicyManager;
 
   @Before
   public void before() throws MuleException {
-    CoreEvent response = testEvent();
-
     extensionModel = mock(ExtensionModel.class);
     when(extensionModel.getXmlDslModel()).thenReturn(XmlDslModel.builder().setPrefix("mock").build());
 
     componentModel = mock(ComponentModel.class, withSettings().extraInterfaces(EnrichableModel.class));
     when((componentModel).getModelProperty(CompletableComponentExecutorModelProperty.class))
-        .thenReturn(of(new CompletableComponentExecutorModelProperty((cp, p) -> (ctx, callback) -> callback.complete(response))));
+        .thenReturn(of(new CompletableComponentExecutorModelProperty(IdentityExecutor::create)));
```

Comment:
This can no longer respond with a cached `Event` instance (the result of `testEvent()` [is cached](https://github.com/mulesoft/mule/blob/8a03b5b3f13427d548ba644a96408939539577c8/tests/unit/src/main/java/org/mule/tck/junit4/AbstractMuleTestCase.java#L331)).

This is because the response is modified (for example by adding/removing the SDK internal context) and because now some tests will be feeding events concurrently through the `ComponentMessageProcessor`, it will be causing concurrent modifications of the same instance (instance that is not thread-safe).

We will be now using different Event instances as the input for all tests in this TestCase. And for this particular processor, we will be returning the input event untouched. It didn't seem to increase the total duration of this TestCase by any noticeable amount.